src/Stratis.Features.FederatedPeg/Stratis.Features.FederatedPeg.csproj

```diff
@@ -22,6 +22,7 @@
     <ProjectReference Include="..\Stratis.Bitcoin.Features.Notifications\Stratis.Bitcoin.Features.Notifications.csproj" />
     <ProjectReference Include="..\Stratis.Bitcoin.Features.SmartContracts\Stratis.Bitcoin.Features.SmartContracts.csproj" />
     <ProjectReference Include="..\Stratis.Features.Collateral\Stratis.Features.Collateral.csproj" />
+    <ProjectReference Include="..\Stratis.Features.Diagnostic\Stratis.Features.Diagnostic.csproj" />
```

Comment:
We shouldnt be adding this project reference. Im not sure what the purpose is of the `Diagnostic` project anymore anyway.

Reply:
Removed.